prov/rxm/src/rxm_ep.c

```diff
@@ -2425,6 +2426,8 @@ static int rxm_ep_msg_res_open(struct rxm_ep *rxm_ep)
 		else
 			((struct sockaddr_in6 *)(rxm_ep->msg_info->src_addr))->sin6_port = 0;
 	}
+	/* Setting this here would help avoid setting it on every MSG endpoint open */
+	rxm_ep->msg_info->dest_addrlen = rxm_ep->msg_info->src_addrlen;
```

Comment:
Unless the dest_addr is non-null, I would not set this here.  This could cause confusion.

Reply:
Hmm, good point. I'll move it to where we need it then.

Reply:
How about we just set the dest_addr same as src_addr here?

Reply:
I would leave it unset if the correct value can't be used, or sometime down the road some change might use it and try to figure out why it's not working.

Reply:
The stand-alone setting of dest_addrlen has been removed.